kibble/models/site_brand_test.go

```diff
@@ -0,0 +1,158 @@
+//    Copyright 2018 SHIFT72
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetImageOnEmpty(t *testing.T) {
+
+	var site Site
+
+	path := site.SiteBrand.GetImage("logo@1x", "/defaultPath")
+	assert.Equal(t, string(path), "/defaultPath")
+
+}
+
+func TestGetLinkOnEmpty(t *testing.T) {
+
+	var site Site
+
+	path := site.SiteBrand.GetLink("css", "/defaultPath")
+	assert.Equal(t, string(path), "/defaultPath")
+
+}
+
+func TestGetLinkCSS(t *testing.T) {
```

Comment:
Yeah I thought that was the case, however I did just make the tests match the ACs

Reply:
I reckon remove them.  The unit tests don't need to match the ACs because the bit of code you are testing here has no knowledge of the ACs or the different image types... you only need to test that it returns the default if nothing is found, or returns the URL from the map if it is found... doesn't matter what the data actually is.

Reply:
I left the two top ones testing completely empty info there, as I think having a Site variable with no initialized data, but still showing the function to be working properly is nice, but can remove, I have removed all tests after the test that starts on L58